### PR TITLE
[tommy] Added rule for trailing newlines

### PR DESCRIPTION
**Pull request by tommydangerous**
PR: https://git.musta.ch/airbnb/airbnb/pull/36236/files
priority: low
size: small

Contents:
- Add some rules for using newlines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +163,76 @@ Airbnb also maintains a [JavaScript Style Guide][airbnb-javascript].
     robot.add_trait(:human_like_intelligence)
     ```
 
+### Trailing
+* Do not add more than 1 newline between code
```
Comment (BMorearty) [inline README.md]: Use a period at the end of a sentence.

Comment (tommydangerous) [inline README.md]: Fixing

Comment (BMorearty) [inline README.md]: Also, what does "between code" mean?

Perhaps you meant "between methods?"

File: README.md [hunk 2]
```diff
@@ -162,6 +163,76 @@ Airbnb also maintains a [JavaScript Style Guide][airbnb-javascript].
...
+
+  # good
+  def bow
+  end
+
+  def arrow
+  end
+  ```
+
+* Avoid adding newlines between comments
```
Comment (BMorearty) [inline README.md]: Use a period at the end of a sentence.

**Comment by BMorearty** [comment]
I like most of these.

But "avoid newlines between comments" seems so blindingly obvious that the only time I could see someone putting a newline between comments is when they are separate comments. And in that case,  you want a newline. So I would remove that guideline.

File: README.md (hunk 1 above)
Comment (tommydangerous) [inline README.md]: K, fixing

**Comment by tommydangerous** [comment]
Okay, removing that `comment` section

**Comment by ljharb** [comment]
:+1:

**Comment by BMorearty** [comment]
@tommydangerous Please give people more than half an hour to see these PRs before you merge.

**Comment by tommydangerous** [comment]
Reverting

**Comment by tommydangerous** [comment]
Okay, I reverted. Sorry about that.

**Comment by venantius** [comment]
In addition to blocks, methods, etc. I'd also like to see classes and modules there (per PRs 37939 and 37942) in monorail.

**Comment by venantius** [comment]
(but otherwise this gets high level approval from me :))

**Pull request merged**